app/components/ItuneCard/tests/index.test.tsx

```diff
@@ -27,5 +31,18 @@ describe('<ItuneCard />', () => {
     expect(baseElement).toMatchSnapshot();
     expect(screen.queryByRole('img')).toBe(null);
     expect(screen.queryByRole('Yung Xiety')).toBe(null);
+    expect(screen.queryByRole('audio')).toBe(null);
+  });
+
+  it('should call the handleOnPlay function and play song', () => {
+    const { getByTestId } = render(<ItuneCard {...song} currentTrackId={1} handleOnPlay={submitSpy} />);
+    fireEvent.play(getByTestId('audio-element'));
+    expect(submitSpy).toBeCalledWith(1);
+  });
+
+  it('should call the handleOnPlay function and pause song', () => {
+    const { getByTestId } = render(<ItuneCard {...song} currentTrackId={1} handleOnPlay={submitSpy} />);
+    fireEvent.pause(getByTestId('audio-element'));
+    expect(submitSpy).not.toBeCalledWith(0);
```

Comment:
We can improve this assertion, it is too confusing. Could you please simplify it ? What happens in `ITuneCard` when you pause a song ?